peripherals/tim/tim_l0.yaml

```diff
@@ -30,8 +30,5 @@
       LSE: [5, "TIM2 ETR input is connected to LSE"]
       HSI: [3, "TIM2 ETR input is connected to HSI16 when HSI16OUTEN bit is set"]
 
-"TIM21":
-  OR:
-    TI2_RMP:
-      GPIO: [0, "TIM2x TI2 input connected to GPIO"]
-      COMP2_OUT: [1, "TIM2x TI2 input connected to COMP2_OUT"] 
```

Comment:
I'm not really sure tbh, but it doesn't make a material difference as `tim21.yaml` is only used in `tim_l0.yaml. I guess it could cause issues in the future if another config uses `tim21.yaml`, but the `OR` definition in `tim21.yaml` looks more complete anyway.

Reply:
Ah, I see what you mean now. Fixed with another commit.